src/Evolution/Initialization/SetVariables.hpp

```diff
@@ -52,8 +52,7 @@ namespace Actions {
 ///   * System::primitive_variables_tag (if system has primitive variables)
```

Comment:
Can you make these types of commit messages more specific? Eg `Rename initialization_option_tags in SetVariables.hpp` so that it's easier to figure out if the commit in the history could be responsible for a bug